docs/content/developer/iota-identity/how-tos/decentralized-identifiers/create.mdx

```diff
@@ -0,0 +1,115 @@
+---
+title: Creating a Decentralized Identity
+sidebar_label: Create and Publish
+description: Create DID Documents and publish them to an IOTA network.
+image: /img/identity/icon.png
+tags:
+    - how-to
+    - identity
+    - did
+---
+
+
+If you want to benefit from Self-Sovereign Identity,
+you need to create a [Decentralized Identity](../../explanations/decentralized-identifiers.mdx).
+This identity consists of many parts that have different functions.
+
+:::note DID method Specification
+
+Note that the Iota Identity Framework follows [IOTA DID Method Specification](../../../../references/iota-identity/iota-did-method-spec.mdx).
+
+:::
+
+## Identity Generation Process
+
+### 1. Get Funds to cover the gas costs
+
+The first thing you will need to generate an identity is an address with enough funds to cover
+the gas cost of the whole procedure.
+In test networks as well as local ones, you can use a faucet to request funds.
+
+:::tip
+
+If you want to use the main IOTA network,
+you will need an an address with actual IOTA funds to create a new Identity.
+
+:::
+
+<div className={'hide-code-block-extras'}>
+<Tabs groupId="language" queryString>
+<TabItem value="rust" label="Rust">
+
+```rust reference
```

Comment:
we changed the import strategy for the move docs, so these reference imports aren't working 

![image](https://github.com/user-attachments/assets/0c186c97-0ea2-465a-8202-566886ce6460)

Reply:
https://github.com/saucelabs/docusaurus-theme-github-codeblock

Reply:
what is required to fix it?